lib/campaign/CampaignState.cpp

```diff
@@ -81,13 +81,14 @@ ImagePath CampaignRegions::getNameFor(CampaignScenarioID which, int colorIndex,
 {
 	auto const & region = regions[static_cast<int>(which)];
 
-	static const std::string colors[2][8] =
+	static const std::string colors[3][8] =
 	{
+{"", "", "", "", "", "", "", ""},
```

Comment:
1. Indent?
2. Ideally should be somewhere in json file

Reply:
The colorcode with 0 characters :)

The Chronicles campaign symbols don't have colors, so no colorname in files. :)